jetty-ee9/jetty-ee9-servlet/pom.xml

```diff
@@ -73,7 +73,7 @@
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
           <!-- Reuse Forks causes test failures -->
-          <reuseForks>false</reuseForks>
+          <reuseForks>true</reuseForks>
```

Comment:
You can just delete this line (and the comment), the default is true anyway, and it was only added to workaround the test failure at the time.